test-cases/longevity/longevity-10gb-3h.yaml

```diff
@@ -21,4 +21,10 @@ space_node_threshold: 64424
 
 gce_n_local_ssd_disk_db: 2
 use_preinstalled_scylla: true
-run_db_node_benchmarks: true
+
+server_encrypt: true
```

Comment:
Are you sure that this commit must be in the PR?

Reply:
I need to remove it, already tested what needed with it

Reply:
removed